roles/rsyslog/defaults/main.yaml

```diff
@@ -291,60 +286,29 @@ rsyslog_weight_map:
   'input': '90'
   'inputs': '90'
 
-# rsyslog_rules
-#
-# List of YAML dictionaries, each dictionary should contain ``rsyslogd``
-# configuration in a special format. See :ref:`rsyslog_rules` for more
-# details. This list should be used for configuration of all hosts in the
-# inventory.
-rsyslog_rules: []
-
-# rsyslog_group_rules
-#
-# List of YAML dictionaries, each dictionary should contain ``rsyslogd``
-# configuration in a special format. See :ref:`rsyslog_rules` for more
-# details. This list should be used for configuration of a group of hosts in
-# the inventory.
-rsyslog_group_rules: []
-
-# rsyslog_host_rules
-#
-# List of YAML dictionaries, each dictionary should contain ``rsyslogd``
-# configuration in a special format. See :ref:`rsyslog_rules` for more
-# details. This list should be used for configuration of specific hosts in the
-# inventory.
-rsyslog_host_rules: []
-
-# rsyslog_dependent_rules
-#
-# List of YAML dictionaries, each dictionary should contain ``rsyslogd``
-# configuration in a special format. See :ref:`rsyslog_rules` for more
-# details. This list should be used for configuration by other Ansible roles.
-rsyslog_dependent_rules: []
-
-# rsyslog_.*_rules
```

Comment:
how is the deleted block above related to the subject of this PR?

Reply:
Since I found these "rules" are not used anywhere, I cleaned this up.  Do you want me to move this to a new pr?